ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -17,6 +17,7 @@
 		97C146FC1CF9000F007C117D /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FA1CF9000F007C117D /* Main.storyboard */; };
 		97C146FE1CF9000F007C117D /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FD1CF9000F007C117D /* Assets.xcassets */; };
 		97C147011CF9000F007C117D /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FF1CF9000F007C117D /* LaunchScreen.storyboard */; };
+		EE91B99237E91BD5F7ED765C /* Pods_Runner.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 36786F88E51B42C1CE027EAC /* Pods_Runner.framework */; };
```

Comment:
Is this file necessary? It seems to be something that gets built from the user side.

Reply:
Even though it gets generated each time, it's better to keep it there as a base, so if someone doesn't have the minimum version, it would not run unless they have the latest changes on the repo.